charts/patroni/values.yaml

```diff
@@ -25,6 +25,10 @@ credentials: {}
   # appDbName: app
   # # Username of application account
   # appDbUserame: app
+# To use an existing secret instead of having the chart generate one, provide the name of the existing secret here.
+# Note that this secret must be formatted the same way as the secret generated by the chart.
+# Leave it blank to allow the helm chart to generate a secret for you.
+externalCreds: ""
```

Comment:
Technically this behaves more as a secret override than it is an externalCred. May be better to name this `secretOverride` or similar to stick with existing convention like we see in Lines 16 and 17.
We should also change the default value to be a tilde instead as that's the default way of representing a null in YAML.
```yaml
secretOverride: ~
```

Reply:
done :)